core/test/dataset_comparison_test.cpp

```diff
@@ -182,8 +182,9 @@ TEST_F(Dataset_comparison_operators, extra_data) {
 TEST_F(Dataset_comparison_operators, extra_variance) {
   auto extra = dataset;
   extra.setData("val",
-                makeVariable<double>({Dim::X, 4}, std::vector<double>(4),
-                                     std::vector<double>(4)) /*LABEL_1*/);
+                createVariable<double>(Dimensions{Dim::X, 4},
+                                       Values(std::vector<double>(4)),
+                                       Variances(std::vector<double>(4))));
```

Comment:
As above, I think `createVariable<double>(Dims{Dim::X}, Shape{4}, Values(4), Variances(4));` does the job?